homeassistant/components/climate/strings.json

```diff
@@ -151,29 +151,29 @@
     },
     "set_preset_mode": {
       "name": "Set preset mode",
-      "description": "Sets preset mode.",
+      "description": "Sets preset operation mode.",
       "fields": {
         "preset_mode": {
           "name": "Preset mode",
-          "description": "Preset mode."
+          "description": "Preset operation mode."
         }
       }
     },
     "set_temperature": {
       "name": "Set target temperature",
-      "description": "Sets target temperature.",
+      "description": "Sets the target temperature for a climate device.",
```

Comment:
"for a climate device" is redundant already at this point; we should remove that.

Same for the other spots this was added.